cmd/repl/repl.go

```diff
@@ -64,6 +68,57 @@ func header(head map[string]interface{}) ([]string, error) {
 	return cols, nil
 }
 
+func imageCat(s string, isTerminal bool) bool {
+	if !(isTerminal && strings.HasPrefix(s, "<div")) {
+		return false
+	}
+	match := regexp.MustCompile(`base64,(.*)'`).FindStringSubmatch(s)
+	if len(match) == 2 {
+		image, e := base64.StdEncoding.DecodeString(match[1])
+		if e != nil {
+			return false
+		}
+		if it2Check == false {
+			fmt.Println("data:text/html,", s)
+			fmt.Println()
+			fmt.Println("To view the image, paste the above url to a web browser or use iTerm2 as terminal.")
+			return true
+		}
+
+		tmpfile, err := ioutil.TempFile("/tmp", "sqlflow")
+		if err != nil {
+			return false
+		}
+		defer os.Remove(tmpfile.Name()) // clean up
+
+		if _, err := tmpfile.Write(image); err != nil {
+			tmpfile.Close()
+			return false
+		}
+		if err := tmpfile.Close(); err != nil {
+			return false
+		}
+		cmd := exec.Command("gosr", tmpfile.Name())
```

Comment:
It seems that we can call https://godoc.org/github.com/mattn/go-sixel#Decoder instead of running a sub-process. Otherwise, we'd have to require our users to install gosr.

Reply:
A user can only run REPL in a Docker container at the moment, so she doesn't have to install gosr. Anyway, it's great not to have to depend on another CLI tool, I'll fix this asap.